### PR TITLE
Log acc dist data

### PR DESCRIPTION
**Pull request by avsek477**
- Log different event datas for acc/dist algo
- Issue fix for `last trade price` event 
- Update log writer to create folder according to gid 

Task: https://app.asana.com/0/1199196876170724/1200420964765198

### CODE REVIEW

**Review by vitaliystoliarovcc — APPROVED** [review]
lgtm

File: lib/accumulate_distribute/events/data_managed_candles.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ const hasIndicatorOffset = require('../util/has_indicator_offset')
 const hasIndicatorCap = require('../util/has_indicator_cap')
 const parseChannelKey = require('../../util/parse_channel_key')
 
+const filterCandleData = (candle, indicatorType, indicatorTypeArgs, indicator, candleIgnored = false) => {
+  return [indicatorType, new Date(candle.mts), candle[indicatorTypeArgs.candlePrice], indicatorTypeArgs.candlePrice, indicator.getName(), indicator.v(), +candleIgnored]
```
Comment (robertkowalski) [inline lib/accumulate_distribute/events/data_managed_candles.js:9]: line length should be max 80

File: lib/accumulate_distribute/util/get_order_log_data.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const { cid, type, amount: orderAmount, price: orderPrice } = order
+
+  data[orderMapIndex.cid] = cid
+  data[orderMapIndex.type] = type
+  data[orderMapIndex.orderAmount] = orderAmount
+  data[orderMapIndex.orderPrice] = orderPrice
+
+  if (hasOBRequirement(args) && lastBook) {
+    const [topAskOrderId = null] = lastBook.topAskLevel() || []
+    const [topBidOrderId = null] = lastBook.topBidLevel() || []
```
Comment (robertkowalski) [inline lib/accumulate_distribute/util/get_order_log_data.js:42]: why `[topBidOrderId = null]` AND `|| []`

**Review by robertkowalski — CHANGES_REQUESTED** [review]
lgtm, two small nits, then ready for merge

File: lib/accumulate_distribute/events/data_managed_candles.js (hunk 1 above)
Comment (avsek477) [reply lib/accumulate_distribute/events/data_managed_candles.js]: fixed.

File: lib/accumulate_distribute/util/get_order_log_data.js (hunk 2 above)
Comment (avsek477) [reply lib/accumulate_distribute/util/get_order_log_data.js:42]: https://github.com/bitfinexcom/bfx-api-node-models/blob/e10007c7a78a46ad1ccefc806b2cd43ef54c6d64/lib/order_book.js#L319

- destructing for 0 index element for the order id. so, [topAskOrderId = null]
- topAskLevel() might return null or an array. so, `lastBook.topAskLevel() || []`

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**